From 5352d5128e3a2b7eac843bd24f7d1f8c6a5823f1 Mon Sep 17 00:00:00 2001
From: Ezra Buehler <ezra.buehler@husqvarnagroup.com>
Date: Thu, 19 Sep 2024 18:29:41 +0200
Subject: [PATCH] arm: at91: gardena-smart-gateway-at91sam: Enable boot counter

The boot count/limit feature is needed in order to implement a fallback
mechanism that is triggered when a system update goes wrong.

Signed-off-by: Ezra Buehler <ezra.buehler@husqvarnagroup.com>
Upstream-Status: Pending
---
 configs/gardena-smart-gateway-at91sam_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/configs/gardena-smart-gateway-at91sam_defconfig b/configs/gardena-smart-gateway-at91sam_defconfig
index 439fcc057539..fffa53fd8caa 100644
--- a/configs/gardena-smart-gateway-at91sam_defconfig
+++ b/configs/gardena-smart-gateway-at91sam_defconfig
@@ -118,3 +118,5 @@ CONFIG_WDT=y
 CONFIG_WDT_AT91=y
 # CONFIG_UBIFS_SILENCE_MSG is not set
 CONFIG_LZMA=y
+CONFIG_CMD_BOOTCOUNT=y
+CONFIG_BOOTCOUNT_LIMIT=y
-- 
2.43.0

